app/views/api_keys/new.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with url: api_keys_create_path, method: :post, local: true do %>
```

Comment:
I'm wondering whether we should just use the govuk formbuilder https://govuk-form-builder.netlify.app/?